test/integration/api/forms/forms.js

```diff
@@ -733,7 +734,7 @@ describe('api: /projects/:id/forms (create, read, update)', () => {
             .send(readFileSync(appRoot + '/test/data/simple.xlsx'))
             .set('Content-Type', 'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet')
             .expect(200)
-            .then(() => asAlice.get('/v1/projects/1/forms/simple2.xls')
+            .then(() => asAlice.get('/v1/projects/1/forms/simple2/draft.xls')
```

Comment:
I tried reverting this line too and the test still seemed to run.

Reply:
:+1: reverted.